### DISCUSSION TITLE
AI

### DISCUSSION THREAD

**Discussion by Madib036** [community/community#138637] [Codespaces] [answered]
### Select Topic Area

Show & Tell

### Body

Who do you think creates and design the AI Models? Computers, humans, or the government

**Comment by Akash1134** [comment, marked as answer]
Hi @Madib036 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
